benchmarks/helics/PholdFederate.hpp

```diff
@@ -26,21 +26,29 @@ class PholdFederate
 
     int index_ = 0;
     int maxIndex_ = 0;
-    int initEvCount_ = 16; // starting number of events
+    unsigned int initEvCount_ = 16; // starting number of events
     double localProbability_ = .1; // probability of local events
```

Comment:
did Peter recommend this value,  might want a comment on why these values were chosen

Reply:
The values are vaguely based on a paper for the Time Warp optimistic synchronization protocol for PDES.